examples/stm32/nucleo-h723zg-make-baremetal-builtin/mongoose_custom.h

```diff
@@ -0,0 +1,13 @@
+#pragma once
+
+#define MG_ARCH MG_ARCH_NEWLIB
+#define MG_OTA MG_OTA_FLASH
+#define MG_DEVICE MG_DEVICE_STM32H7
+#define MG_DEVICE_SINGLE_BANK
```

Comment:
All our defines are numeric - please follow the tradition. Maybe it is better to use DUAL_BANK? Cause single bank is kind of a default ? I'd add this to the config.h:

```c
#ifndef MG_DEVICE_DUAL_BANK
#define MG_DEVICE_DUAL_BANK 0
#endif
```

Then for H743 and H563 we will add this:

```c
#define MG_DEVICE_DUAL_BANK 1
```